roles/containers/defaults/main.yml

```diff
@@ -2,14 +2,14 @@
 docker_network_name: trentonet
 web_container_image: ghcr.io/trento-project/trento-web:rolling
 web_container_name: trento_web
-web_container_port: 4000
+web_container_port: "{{ 32767 | random(start=1024, seed=trento_server_name) }}"
```

Comment:
Add random generation of container ports.
With this method, we have a predictable way of creating ports based on the `trento_server_name`.
As our PR generator should use a different server name always, we shouldn't have collisions, but as always the numbers are finite...
In any case, the user can still configure custom values setting these 2 variables in their vars files and overwrite them